Sources/TuistKit/ProjectEditor/ProjectEditorMapper.swift

```diff
@@ -41,6 +45,12 @@ final class ProjectEditorMapper: ProjectEditorMapping {
         if !templates.isEmpty {
             manifestsDependencies.append(.target(name: "Templates"))
         }
+        if setupPath != nil {
```

Comment:
Why do the manifest targets depend on the targets `Setup`, `Config`, and `Templates`? I think that's not necessary cc @fortmarek

Reply:
Huh, yeah, they probably should not, good catch (it's enough they are added to `targets` to `Project` which is then generated)

Reply:
I have changed it. Please verify it.